bzl/obazl.edn

```diff
@@ -0,0 +1,62 @@
+;; use with `$ obazl ingest`
+{
+ :workspace "ocaml-extlib"
+ :config-pkg "//bzl/config"
+ ;; :ppx {:pkg "//ppx"
+ ;;       :exec "foo"}
+ :externals { }
+ :exclusions #{"src/configure.ml"}
+ :archives [{:name ocaml-extlib
+             :components [extBytes enum bitSet dynArray
+                          extArray extHashtbl extList extString extBuffer
+                          global IO option pMap std
+                          uChar uTF8 unzip refList optParse
+                          dllist extLib]
+             }
+            ]
+ :config-flags {:always-link {:default false :config-name "always_link" :flag "-linkall"}
+                :cmt {:default true :config-name "output_cmt" :flag "-bin-annot"}
+                :debug {:default true :flag "-g"}
+                :verbose {:default false :flag "-verbose"}}
+ :compile-options ["-strict-formats"
+                   "-short-paths"
+                   "-w"
+                   "+A-42-48"
+                   "-warn-error"
+                   "+A-3-44"]
+ :cppo {:defines {;; :with-bytes {:define "WITH_BYTES" :flag true}
```

Comment:
Nit: this comment seems like it should be removed

Reply:
That file is used by the obazl conversion tool; specifically it configures the `ingest` command, which is for repos/packages that do not have Dune files.  Since this is the only such repo (I think) in Mina, I only implemented enough to get this repo out the door, so that comment is a memo to self.  Once I get the rest of Mina out the door, I'll circle back and finish the implementation and remove this comment.  I'd put it somewhere else, but then I'd be sure to forget where it is. ;)